package.json

```diff
@@ -2,46 +2,103 @@
   "name": "videojs-contrib-media-sources",
   "version": "2.4.3",
   "description": "A Media Source Extensions plugin for video.js",
-  "main": "videojs-media-sources.js",
+  "author": "Brightcove",
+  "license": "Apache-2.0",
+  "main": "src/plugin.js",
```

Comment:
Not only that but browserify transforms don't run on dependencies and we don't want to force users of the package to use babel themselves.